### PR TITLE
Rebalances klutz

### PR DESCRIPTION
**Pull request by Guidesu**
## About The Pull Request
Klutz is a horrible perk based on pure RNG and being utterly awful to have.
So I toned it down meaning that people will not have much of trouble of having it but still a decent chance of failing of doing certain actions.
Also, re-balances the klutz Upbringing.
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:
add: Klutz upbringing has their stats balanced, and they can perform way more task without having a ridiculous chancing of fail.

/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/shields.dm [hunk 1]
```diff
@@ -549,9 +549,10 @@
 	return base_block_chance
 
 /obj/item/shield/buckler/energy/attack_self(mob/living/user as mob)
-	if ((CLUMSY in user.mutations) && prob(50))
-		to_chat(user, SPAN_WARNING("You beat yourself in the head with [src]."))
-		user.take_organ_damage(5)
+	if ((CLUMSY in user.mutations) && prob(15))
+		to_chat(user, SPAN_WARNING("You accidentally bash yourself with the [src]."))
+		user.damage_through_armor(10, BURN, user.hand)
+		user.Weaken(3 * force)
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/shields.dm:555]:
```suggestion
		user.Weaken(1 * force)
```

**Comment by Trilbyspaceclone** [comment]
I luv it!

File: code/datums/setup_option/backgrounds/origin_upbringing.dm [hunk 2]
```diff
@@ -89,18 +89,18 @@
...
 	name = "Klutz"
-	desc = "Your entire life has been a series of unlucky and often self inflicted accidents, you spent enough time hurting yourself due to your own clumsiness that you've built up a bit more \
-	pain tolerance than most. A shame this doesn't stop you from failing even the most basic tasks at times. In fact, you find some tasks that require precision damn near impossible."
+	desc = "Your entire life has been a series of unlucky and often self-inflicted accidents, you spent enough time hurting yourself due to your own clumsiness that you've built up a more \
+	pain tolerance than most of common folks, meaning that due to this clumsiness problem, you were always a good fighter due to your natural resistance. A shame, this doesn't stop you from time to time failing even the most basic tasks at times. In fact, you find some tasks that require precision damn near impossible, and even handling or using guns are literally impossible to use. And due to this problem, you have a serious problem of perception around, and you are not the most smart guy around here."
 
-	perks = list(/datum/perk/klutz)
+	perks = list(/datum/perk/klutz,/datum/perk/oddity/harden)
```
Comment (Trilbyspaceclone) [inline code/datums/setup_option/backgrounds/origin_upbringing.dm:95]:
```suggestion
	perks = list(/datum/perk/klutz)
```
no perk plz

Comment (Guidesu) [reply code/datums/setup_option/backgrounds/origin_upbringing.dm]: okay

**Comment by cdb-is-not-good** [comment]
These numbers are....a bit too high. Like, insanely so. This is a straight buff to klutz and looks more like it'll become the powergame special seeing as we already have. Large number of people who stand to benefit from only using melee(which only works because of a quirk of the code.).

**Comment by Guidesu** [comment]
> These numbers are....a bit too high. Like, insanely so. This is a straight buff to klutz and looks more like it'll become the powergame special seeing as we already have. Large number of people who stand to benefit from only using melee(which only works because of a quirk of the code.).

any suggestion then?

**Comment by Kazkin** [comment]
Have to agree with cbd, these numbers are insanely good and this makes klutz a near non-issue for almost everything that before was huge detrimental.

Stat wise I'd say:
10 RB
20 TGH
-20 VIG
0 BIO
0 MEC
0 Cog

I can almost agree with most of the other changes, but I'd rather have guns have a 15% chance of shooting the self rather then disable them entirely. This makes klutz a bit more of a game as you make guns risky but still get that sweet ROB/TGH bonus. Additionally, I'd greatly lean towards adding klutz checks to various other things, such as mechs, syringes, bandages, mops, soaps, etc. etc.

**Comment by Guidesu** [comment]
> Have to agree with cbd, these numbers are insanely good and this makes klutz a near non-issue for almost everything that before was huge detrimental.
> 
> Stat wise I'd say: 10 RB 20 TGH -20 VIG 0 BIO 0 MEC 0 Cog
> 
> I can almost agree with most of the other changes, but I'd rather have guns have a 15% chance of shooting the self rather then disable them entirely. This makes klutz a bit more of a game as you make guns risky but still get that sweet ROB/TGH bonus. Additionally, I'd greatly lean towards adding klutz checks to various other things, such as mechs, syringes, bandages, mops, soaps, etc. etc.

Okay! I wil change it

**Comment by Guidesu** [comment]
> 

will also do later don't worries

**Comment by Guidesu** [comment]
Done with all the request

**Pull request merged**